presto-docs/src/main/sphinx/release/release-0.273.rst

```diff
@@ -0,0 +1,75 @@
+=============
+Release 0.273
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix Disaggregated Coordinator resource group aggregation issue related to WAITING_FOR_PREREQUISITE queries.
```

Comment:
- lower case 'coordinator'?
- can we explain what is the exact issue exposed to the users?

Reply:
done and added some details about the issue.